docs/RELEASING.md

```diff
@@ -33,28 +33,28 @@ tf-encrypted below:
    master for the given version in setup.py (e.g. `git tag 0.1.0-rc0`) and push
    to GitHub (e.g. `git push origin 0.1.0-rc0`). This will trigger a deploy on
    Circle CI which will push the artifacts to PyPI and Docker Hub.
-3. Once the build on Circle CI has passed create pull down the library from
+3. Once the build on Circle CI has passed, pull down the library from
    pypi and ensure the different examples in our library and documentation work
    as expected. If at any point a bug is found repeat steps 1-3 until the
    release works as expected.
 4. Once the release candidate work as expected, create a new branch based off
    master (e.g. `release-0.1.0`), update the files above with the true release
-   version, and merge it into the master branch on TF Encrypted's GitHub. Once
-   you have verified that the CircleCI jobs have finished as we did for the
-   release candidate, create a tag (e.g. `git tag 0.1.0`) off of the
-   merge commit and push it to github (e.g. `git push origin 0.1.0`).
+   version, and merge it into the master branch. Once you have verified that 
```

Comment:
I mistakenly merged to local master branch while reading this the first time, so maybe it’s worth specifying that this is the remote master branch

Reply:
Alright, changing